PWGLF/TableProducer/converters/strarawcentsconverter.cxx

```diff
@@ -16,17 +16,50 @@
 using namespace o2;
 using namespace o2::framework;
 
+// Converts V0 version 001 to 002
 struct strarawcentsconverter {
+  Produces<aod::DauTrackExtras_001> dauTrackExtras_001;
+  Produces<aod::V0MCCores_001> v0MCCores_001;
   Produces<aod::StraRawCents_001> straRawCents_001;
   Produces<aod::StraRawCents_003> straRawCents_003;
 
-  void processStraRawCents000to001(aod::StraRawCents_000 const& straRawCents_000)
+  void process000to001(aod::StraRawCents_000 const& straRawCents_000, aod::DauTrackExtras_000 const& dauTrackExtras_000, aod::V0MCCores_000 const& v0MCCores_000)
```

Comment:
You should create another converter, I believe, rather than trying to convert different types of tables in only one task. This leads to some issues, as the same task has multiple "Produces" calls and those empty out tables that are not replaced (= if their corresponding process function is disabled). Can you please adjust? Sorry for the trouble, thanks!